client/my-sites/stats/site.jsx

```diff
@@ -162,7 +163,7 @@ class StatsSite extends Component {
 					slug={ slug }
 				/>
 				<div id="my-stats-content">
-					<StatsBanners siteId={ siteId } slug={ slug } />
+					{ ! isCustomerHomeEnabled && <StatsBanners siteId={ siteId } slug={ slug } /> }
```

Comment:
Adding a comment here that `StatsBanners` are shown on the customer home instead, would make it easier to understand what's going on.